app/controllers/user.js

```diff
@@ -31,3 +33,22 @@ exports.signIn = (req, res, next) =>
       }
     })
     .catch(next);
+
+exports.getAllUsers = (req, res, next) => {
+  const limit = parseInt(req.query.limit) || 10;
+  const page = parseInt(req.query.page) || 1;
+
+  const paginationParams = {
+    limit,
+    page,
+    offset: (page - 1) * limit,
+    orderBy: req.query.orderBy || defaultOrderBy,
+    order: req.query.order || ascOrder
+  };
+  return userService
+    .findAllPagination(paginationParams)
+    .then(users => {
+      res.send({ users });
```

Comment:
Nested user key `users` is ugly for the response, client would receive something like this:
``` 
{
  users: {
    users: [...],
    pages: ....,
    etc
  }
}
``` 
instead try return something like this:
```
{
    users: [...],
    pages: ....,
    etc
}
```